docset/winserver2022-ps/remoteaccess/Get-RemoteAccessRoutingDomain.md

```diff
@@ -26,8 +26,11 @@ The **Get-RemoteAccessRoutingDomain** cmdlet retrieves routing domain configurat
 ## EXAMPLES
 
 ### Example 1: Retrieve configuration for a specific routing domain
+```powershell
```

Comment:
We always want to have a blank line around block elements, like headings and code blocks

````suggestion

```powershell
````